src/dropdown/dropdown.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+import Controller from '../controller.js';
+import Button from '../button/button.js';
+import ControllerCollection from '/ckeditor5/ui/controllercollection.js';
+
+import DropdownButtonView from './dropdownbuttonview.js';
+
+import DropdownPanel from './dropdownpanel.js';
+import DropdownPanelView from './dropdownpanelview.js';
+
+/**
+ * The basic dropdown controller class.
+ *
+ * @memberOf ui.dropdown
+ * @extends ui.Controller
+ */
+
+export default class Dropdown extends Controller {
+	/**
+	 * Creates a Dropdown instance.
+	 *
+	 * @param {utils.Observable} model
+	 * @param {ui.View} view
+	 */
+	constructor( model, view ) {
+		super( model, view );
+
+		this.collections.add( new ControllerCollection( 'dropdown' ) );
```

Comment:
I don't like this collection's name ;|. It's a bit confusing. Do you have some other ideas? Maybe `'main'`? Like in the boxed editor UI.